### PR TITLE
docs: add instructions to get started with Determined locally to README

### PR DESCRIPTION
**Pull request by ahj-determined**
## Description

Currently, it takes new users a long time to get a working installation of Determined. They run into issues like not knowing there are 3 determined pip packages to install, not using a virtual environment, not installing Docker, and not knowing that --no-gpu option exists. We should advertise what the simplest way to get started with Determined is on the README.

[[DET-4712](https://determinedai.atlassian.net/browse/DET-4712)]

## Test Plan

* Tested instructions manually on Ubuntu and mac OSX.

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.

### CODE REVIEW

**Review by determined-dsw — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -23,17 +23,37 @@ To use Determined, you can continue using popular DL frameworks such as
...
+ 
+Follow [these instructions](https://docs.determined.ai/latest/how-to/installation/requirements.html#install-docker) to install and set up docker.
 
-* [Installation Guide](https://docs.determined.ai/latest/how-to/install-main.html)
+ ```bash
+
+# Start a Determined cluster locally
+python3.7 -m venv ~/.virtualenvs/test
+ . ~/.virtualenvs/test/bin/activate
+ pip install determined determined-cli determined-deploy
```
Comment (neilconway) [inline README.md:35]: Is installing `determined` necessary?

File: README.md [hunk 2]
```diff
@@ -23,17 +23,37 @@ To use Determined, you can continue using popular DL frameworks such as
...
+ det-deploy local cluster-up --no-gpu # Access web UI at localhost:8080. By default, "admin" user accepts a blank password.
```
Comment (neilconway) [inline README.md:36]: I might move this comment to a new line (either before or after `det-deploy`).

File: README.md [hunk 3]
```diff
@@ -23,17 +23,37 @@ To use Determined, you can continue using popular DL frameworks such as
...
+ 
+# Navigate to a Determined example
+ git clone https://github.com/determined-ai/determined
+ cd determined/examples/computer_vision/cifar10_pytorch/
+
+# Submit an experiment to Determined
```
Comment (neilconway) [inline README.md:42]: Say what this does -- i.e., trains a single model on a single GPU

File: README.md [hunk 4]
```diff
@@ -23,17 +23,37 @@ To use Determined, you can continue using popular DL frameworks such as
 TensorFlow and PyTorch; you just need to update your model code to integrate
 with the Determined API.
 
-## Installation
+## Try out Determined locally
```
Comment (neilconway) [inline README.md:26]: Can we make the capitalization consistent with the other section headers?

File: README.md (hunk 2 above)
Comment (neilconway) [reply README.md:36]: Also, is the default user named "admin" or "determined"?

**Review by neilconway — COMMENTED** [review]
Thanks for making these improvements!

I wonder if we should also update the top-level docs index page, which has much of the same content. wdyt?

File: README.md [hunk 5]
```diff
@@ -23,17 +23,37 @@ To use Determined, you can continue using popular DL frameworks such as
...
+
+# Start a Determined cluster locally
+python3.7 -m venv ~/.virtualenvs/test
+ . ~/.virtualenvs/test/bin/activate
+ pip install determined determined-cli determined-deploy
+ det-deploy local cluster-up --no-gpu # Access web UI at localhost:8080. By default, "admin" user accepts a blank password.
+ 
+# Navigate to a Determined example
+ git clone https://github.com/determined-ai/determined
+ cd determined/examples/computer_vision/cifar10_pytorch/
```
Comment (neilconway) [inline README.md:40]: Can we remove the trailing slash? :)

**Comment by ahj-determined** [comment]
Thanks for the review @neilconway! Great idea to also add this to the top-level docs index page. I've pushed that update now.

**Comment by neilconway** [comment]
@ahj-determined Is this ready to be merged?
